pyproject.toml

```diff
@@ -1,3 +1,61 @@
+[project]
+name = "launchable"
+authors = [
+    {name = "Launchable, Inc.", email = "info@launchableinc.com"}
+]
+description = "Launchable CLI"
+readme = "README.md"
+license = {text = "Apache Software License v2"}
+requires-python = ">=3.13"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "click>=8.1",
+    "requests>=2.25",
+    "urllib3>=1.26",
+    "junitparser>=2.0.0",
+    "more-itertools>=7.1.0",
+    "python-dateutil",
+    "tabulate",
+]
+dynamic = ["version"]
+
+[project.urls]
+Homepage = "https://launchableinc.com/"
```

Comment:
Don't you change the information in this PR?

Reply:
I will change it in another PR as well as the command name. Thanks for reminding me.